### PR TITLE
[SPARK-41962][MINOR][SQL] Update the order of imports in class SpecificParquetRecordReaderBase

### PR DESCRIPTION
**Pull request by wayneguow**
### What changes were proposed in this pull request?
Update the order of imports in class SpecificParquetRecordReaderBase.

### Why are the changes needed?
Follow the code style.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Passed GA.

### CODE REVIEW

File: sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java [hunk 1]
```diff
@@ -28,12 +28,9 @@
 import java.util.Map;
 import java.util.Set;
 
-import com.google.common.annotations.VisibleForTesting;
-import org.apache.parquet.VersionParser;
-import org.apache.parquet.VersionParser.ParsedVersion;
-import org.apache.parquet.column.page.PageReadStore;
```
Comment (HyukjinKwon) [inline sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java:34]: Can we fix `dev/checkstyle.xml` to enforce this order?

Comment (wayneguow) [reply sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java:34]: Ok, that's better, and once and for all. I'll make some changes later.

Comment (wayneguow) [reply sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java:34]: After pr #35945 and discussion in pr #35970, the `ImportOrder` todo has been removed, the conclusion is that it's not worth changing all unordered files.
So shall we just make a little change for closing the related issued opened by uses?

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Comment by cloud-fan** [comment]
no need to take this change

**Comment by dongjoon-hyun** [comment]
Hi, @wayneguow and @HyukjinKwon  This broke branch-3.2.
- https://github.com/apache/spark/commits/branch-3.2

**Comment by dongjoon-hyun** [comment]
I reverted the commit because the `import` statement is unused.
- https://github.com/apache/spark/commit/c8b47ec2d0a066f539a98488502fce99efe006f0

**Comment by wayneguow** [comment]
> I reverted the commit from `branch-3.2` because the `import` statement is unused.
> 
> * [c8b47ec](https://github.com/apache/spark/commit/c8b47ec2d0a066f539a98488502fce99efe006f0)
> 
> Screenshot 2023-02-08 at 1 34 13 PM

@dongjoon-hyun  Thank you for resolving it.

**Comment by dongjoon-hyun** [comment]
You can make a new backporting PR to branch-3.2 if needed, @wayneguow .

**Comment by dongjoon-hyun** [comment]
Also, cc @parthchandra for branch-3.2

**Comment by wayneguow** [comment]
> You can make a new backporting PR to branch-3.2 if needed, @wayneguow .

image
It seems that it is ok without those unordered imports in branch-3.2, so just reverting this for branch-3.2 is enough.

**Comment by dongjoon-hyun** [comment]
Thank you for the confirmation, @wayneguow .

**Pull request closed without merging**